src/constants.js

```diff
@@ -13,4 +13,12 @@ export const envs = {
 
 // Variables
 export const prefix = '++';
-export const validCmds = ['ama', 'faq', 'pins', 'salary', 'settings', 'perms'];
+export const validCmds = [
+  'ama',
+  'faq',
+  'pins',
+  'ping',
+  'salary',
+  'settings',
+  'perms'
```

Comment:
unrelated to your specific changes, but can we just remove perms? we don't actually need it